crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -933,10 +934,9 @@ end
 #! Where:
 #! - acct_nonce is the account nonce.
 export.get_acct_nonce
-    padw
     exec.get_current_account_data_ptr push.ACCT_ID_AND_NONCE_OFFSET add
-    mem_loadw
-    movdn.3 drop drop drop
+    add.3
+    mem_load
```

Comment:
Similar comment here as well.

Reply:
thanks 👍🏼  I addressed both in [chore: define a const for use in other procs](https://github.com/0xMiden/miden-base/pull/1585/commits/30d16831abbdfb16b770a723e224dbff9dffa0b2)